<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MRSettingsDialog</class>
 <widget class="QWidget" name="MRSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>581</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Mapnik Renderer Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Uses a PostGreSQL GIS database to get the data and renders it with Mapnik, the official rendering tool for OpenStreetMap. Prerenders the complete map in small tiles for all zoom levels and saves them in png files. When increasing Tile Size and Meta Tile Size take caution of rapidly increasing Memory Consumption. Very Slow preprocessing, large space consumption and requires a working PostGreSQL Database with Openstreetmap Data.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>495</width>
        <height>336</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Files &amp;&amp; Directories</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,1,0">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>DeJaVu Fonts</string>
         </property>
         <property name="buddy">
          <cstring>fontEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="fontEdit"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="fontBrowse">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Mapnik Importer Modules</string>
         </property>
         <property name="buddy">
          <cstring>modulesEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="modulesEdit"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="modulesBrowse">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Mapnik Theme (osm.xml)</string>
         </property>
         <property name="buddy">
          <cstring>themeEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="themeEdit"/>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="themeBrowse">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Browse...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>495</width>
        <height>336</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Zoom Levels</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Render Whole World To</string>
           </property>
           <property name="buddy">
            <cstring>minZoom</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="minZoom">
           <property name="prefix">
            <string>Level </string>
           </property>
           <property name="value">
            <number>6</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Expand Area By</string>
           </property>
           <property name="buddy">
            <cstring>tileMargin</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="tileMargin">
           <property name="suffix">
            <string> Tiles</string>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Zoom Levels</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QCheckBox" name="zoom0">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="zoom5">
            <property name="text">
             <string>5</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="zoom10">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QCheckBox" name="zoom15">
            <property name="text">
             <string>15</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="zoom1">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="zoom6">
            <property name="text">
             <string>6</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="zoom11">
            <property name="text">
             <string>11</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QCheckBox" name="zoom16">
            <property name="text">
             <string>16</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="zoom2">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="zoom7">
            <property name="text">
             <string>7</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="zoom12">
            <property name="text">
             <string>12</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QCheckBox" name="zoom17">
            <property name="text">
             <string>17</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="zoom3">
            <property name="text">
             <string>3</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="zoom8">
            <property name="text">
             <string>8</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="zoom13">
            <property name="text">
             <string>13</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QCheckBox" name="zoom18">
            <property name="text">
             <string>18</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="zoom4">
            <property name="text">
             <string>4</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="zoom9">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>9</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QCheckBox" name="zoom14">
            <property name="text">
             <string>14</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>495</width>
        <height>336</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Tiles</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Tile Size</string>
           </property>
           <property name="buddy">
            <cstring>tileSize</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="tileSize">
           <property name="suffix">
            <string> Pixel</string>
           </property>
           <property name="minimum">
            <number>64</number>
           </property>
           <property name="maximum">
            <number>4096</number>
           </property>
           <property name="singleStep">
            <number>64</number>
           </property>
           <property name="value">
            <number>256</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Margin</string>
           </property>
           <property name="buddy">
            <cstring>margin</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="margin">
           <property name="suffix">
            <string> Pixel</string>
           </property>
           <property name="minimum">
            <number>32</number>
           </property>
           <property name="maximum">
            <number>2048</number>
           </property>
           <property name="singleStep">
            <number>32</number>
           </property>
           <property name="value">
            <number>128</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Meta Tile Size ( Renders X * X Tiles At Once )</string>
           </property>
           <property name="buddy">
            <cstring>metaTileSize</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="metaTileSize">
           <property name="suffix">
            <string> Tiles</string>
           </property>
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>1024</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>495</width>
        <height>336</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Compression</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="colorReduction">
         <property name="text">
          <string>Use 256 Color PNGs</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pngcrush">
         <property name="text">
          <string>Use PNGcrush</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="removeTiles">
         <property name="text">
          <string>Remove Tiles with no routable Edges</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
